mscore/newwizard.cpp

```diff
@@ -150,6 +150,27 @@ TitleWizard::TitleWizard(QWidget* parent)
    : QWidget(parent)
       {
       setupUi(this);
+      connect(this->title, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->subtitle, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->composer, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->poet, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->copyright, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      }
+
+void TitleWizard::characterAfterCursor(int old_pos, int new_pos)
```

Comment:
Is this what we decided would not be necessary with a correctly working version of Qt?  Definitely seems wrong to be needing to be doing this.

Reply:
I think it may be a Qt bug. I had made a new AccessibleLineEdit class but I was getting errors when promoting QLineEdits to use that class. This isn't a part of the masterpalette_navigation [PR](https://github.com/musescore/MuseScore/pull/3344).